cbpi/configFolder.py

```diff
@@ -115,19 +166,26 @@ def create_config_file(self):
         print("Config Folder created")
 
     def create_home_folder_structure(configFolder):
-        pathlib.Path(os.path.join(".", 'logs/sensors')).mkdir(parents=True, exist_ok=True)
+        # pathlib.Path(os.path.join(".", 'logs/sensors')).mkdir(parents=True, exist_ok=True)
```

Comment:
If this line is no longer needed, you should remove it instead of making it a comment. This way it will not confuse future reviewers/developers.